packages/trace-viewer/src/ui/errorsTab.tsx

```diff
@@ -144,7 +65,12 @@ function Error({ message, error, errorId, sdkLanguage, pageSnapshot, revealInSou
     longLocation = stackFrame.file + ':' + stackFrame.line;
   }
 
-  const codeFrame = useCodeFrame(error.stack, sources, 3);
+  const prompt = useAsyncMemo(async () => {
+    if (!error.promptURL)
+      return;
+    const response = await fetch(error.promptURL);
```

Comment:
Why are we fetching it as a URL? Not sure I understand the semantics behind it.

Reply:
That's just how attachments in UI mode work, the attachments tab also fetches contents from a URL.